docs/root/version_history/current.rst

```diff
@@ -9,6 +9,69 @@ Minor Behavior Changes
+* http: allow to use path canonicalizer from `googleurl <https://quiche.googlesource.com/googleurl>`_
+  instead of `//source/common/chromium_url`. The new path canonicalizer is enabled by default. To
+  revert to the legacy path canonicalizer, enable the runtime flag
+  `envoy.reloadable_features.remove_forked_chromium_url`.
+* http: increase the maximum allowed number of initial connection WINDOW_UPDATE frames sent by the peer from 1 to 5.
+* http: no longer adding content-length: 0 for requests which should not have bodies. This behavior can be temporarily reverted by setting `envoy.reloadable_features.dont_add_content_length_for_bodiless_requests` false.
+* http: upstream flood and abuse checks increment the count of opened HTTP/2 streams when Envoy sends
+  initial HEADERS frame for the new stream. Before the counter was incrementred when Envoy received
+  response HEADERS frame with the END_HEADERS flag set from upstream server.
+* lua: added function `timestamp` to provide millisecond resolution timestamps by passing in `EnvoyTimestampResolution.MILLISECOND`.
+* oauth filter: added the optional parameter :ref:`auth_scopes <envoy_v3_api_field_extensions.filters.http.oauth2.v3alpha.OAuth2Config.auth_scopes>` with default value of 'user' if not provided. Enables this value to be overridden in the Authorization request to the OAuth provider.
+* perf: allow reading more bytes per operation from raw sockets to improve performance.
+* router: extended custom date formatting to DOWNSTREAM_PEER_CERT_V_START and DOWNSTREAM_PEER_CERT_V_END when using :ref:`custom request/response header formats <config_http_conn_man_headers_custom_request_headers>`.
+* router: made the path rewrite available without finalizing headers, so the filter could calculate the current value of the final url.
+* tracing: added `upstream_cluster.name` tag that resolves to resolve to :ref:`alt_stat_name <envoy_v3_api_field_config.cluster.v3.Cluster.alt_stat_name>` if provided (and otherwise the cluster name).
+* udp: configuration has been added for :ref:`GRO <envoy_v3_api_field_config.core.v3.UdpSocketConfig.prefer_gro>`
+  which used to be force enabled if the OS supports it. The default is now disabled for server
+  sockets and enabled for client sockets (see the new features section for links).
+* upstream: host weight changes now cause a full load balancer rebuild as opposed to happening
+  atomically inline. This change has been made to support load balancer pre-computation of data
+  structures based on host weight, but may have performance implications if host weight changes
+  are very frequent. This change can be disabled by setting the `envoy.reloadable_features.upstream_host_weight_change_causes_rebuild`
+  feature flag to false. If setting this flag to false is required in a deployment please open an
+  issue against the project.
+
```

Comment:
this got messed up because of yesterday's release -- you'd want to throw everything out except your entry.